erigon-lib/kv/tables.go

```diff
@@ -455,7 +459,7 @@ var ChaindataTables = []string{
 	ActiveValidatorIndicies,
 	EffectiveBalancesDump,
 	BalancesDump,
-
+	// Witnesses,
```

Comment:
table should be initialised? Please add to comment of witness when values in that bucket become obsolete or so. also could be good to add it to some cleanup lists (to have other that rm chaindata options to reset contents of this bcket

Reply:
The table is not used for now.